openhtf/exe/phasemanager.py

```diff
@@ -162,7 +162,7 @@ def _ExecuteOnePhase(self, phase):
         'Executing phase %s with plugs %s', phase.func.__name__, data.plugs)
 
     self._test_state.running_phase = test_record.PhaseRecord(
-        phase.func.__name__, phase.source, docstring=phase.func.__doc__)
+        phase.func.__name__, phase.code_info.sourcecode, docstring=phase.func.__doc__)
```

Comment:
hm, probably better to plumb through all of the CodeInfo attributes, don't you think?  you could even encapsulate all three of these and store a CodeInfo directly on the PhaseRecord if you wanted.